tbx/project_styleguide/templates/patterns/molecules/listing/listing--avatar.html

```diff
@@ -10,16 +10,17 @@ <h2 class="listing-avatar__title listing__title">{{ title }}</h2>
             <div>
                 <p class="listing__details-group">
                     <span class="listing__name mr-1.5">{{ name }}</span>
+                    <span class="listing__job-title">{{ job_title }}</span>
                 </p>
                 <p class="listing__meta">
-                    <span class="mr-1.5">{{ page_type }},</span> {{ reading_time }}
+                    <span class="mr-1.5">{{ date }}{% if reading_time %},{% endif %}</span> {% if reading_time %}{{ reading_time }} min{{ reading_time|pluralize }} read{% endif %}
```

Comment:
Think `date` is missing a value in the pattern library

Reply:
Resolved in 97f3c854c15273d419cd541255168cf81faa0544